src/util/metrics.ts

```diff
@@ -0,0 +1,64 @@
+import express from 'express';
+import { Application, Request, Response } from 'express';
+import client from 'prom-client';
+
+import { Log } from '../logging/Log';
+import { parseArgs } from '../parseArgs';
+
+const { logger } = Log;
+
+const register = new client.Registry();
```

Comment:
These top level calls should be encapsulated in a function and called.  Meaning they should sit in global scope.